src/mev/mev_custom_flood/mev_custom_flood_launcher.star

```diff
@@ -1,6 +1,12 @@
 PYTHON_IMAGE = "ethpandaops/python-web3"
 CUSTOM_FLOOD_SERVICE_NAME = "mev-custom-flood"
 
+# The min/max CPU/memory that mev-custom-flood can use
+MIN_CPU = 10
```

Comment:
where are all of these numbers coming from?

Reply:
yolo